modules/channel-web/src/backend/api.ts

```diff
@@ -297,8 +297,10 @@ export default async (bp: typeof sdk, db: Database) => {
       }
 
       await sendNewMessage(botId, userId, conversationId, payload, req.credentials)
-      await bp.dialog.deleteSession(userId)
-      res.status(200).send({})
+
+      const sessionId = await bp.dialog.createId({ botId, target: userId, threadId: conversationId, channel: 'web' })
```

Comment:
I know you didn't change the logic here, I just don't understand how this works.. you create a new session and delete it ? 

Or is it that `bp.dialog.create` acts like a `getOrCreate` given the params ?

Reply:
It just created the session id from given parameters 
```js
static createIdFromEvent(eventDestination: IO.EventDestination) {
    const { channel, target, threadId } = eventDestination
    return `${channel}::${target}${threadId ? `::${threadId}` : ''}`
  }
```